ingestion/src/metadata/ingestion/source/database/pinotdb/connection.py

```diff
@@ -33,7 +33,7 @@
 
 def get_connection_url(connection: PinotDBConnection) -> str:
     url = get_connection_url_common(connection)
-    url += f"/query/sql?controller={connection.pinotControllerHost}"
+    url += f"/query/sql?controller=http://{connection.pinotControllerHost}/"
```

Comment:
Thinking out loud, could the `http` scheme be passed directly in the input? What would happen if someone uses https?

Reply:
hi @Supan90-Shah3006 LMK if you have any thoughts here. Otherwise I'll revert this line and accept the rest of the docs.

Thanks!

Reply:
```suggestion
    url += f"/query/sql?controller={connection.pinotControllerHost}/"
```